crypto_sign/rainbowIIIc-classic/clean/blas_u32.c

```diff
@@ -0,0 +1,87 @@
+#include "blas_u32.h"
+#include "gf.h"
+
+void PQCLEAN_RAINBOWIIICCLASSIC_CLEAN_gf256v_predicated_add_u32(uint8_t *accu_b, uint8_t predicate, const uint8_t *a, unsigned int _num_byte) {
+    uint32_t pr_u32 = ((uint32_t)0) - ((uint32_t)predicate);
+    uint8_t pr_u8 = pr_u32 & 0xff;
+
+    unsigned int n_u32 = _num_byte >> 2;
+    uint32_t *b_u32 = (uint32_t *)accu_b;
```

Comment:
This is unsafe behaviour if `accu_b` does not meet the `uint32_t` alignment requirements.